mail_tracking/models/mail_composer.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class MailComposer(models.TransientModel):
+    _inherit = 'mail.compose.message'
+
+    no_followers = fields.Boolean(string="Hide follower message",
+                                  default=False)
+
+    @api.multi
+    def send_mail(self, auto_commit=False):
+        res = super().send_mail(auto_commit=auto_commit)
+        message = self.env['mail.message'].browse(
+            self._context.get('message_id'))
+        if message:
```

Comment:
Contrary to what it might seem, `.browse()` doesn't actually call the DB backend. It does that lazily. You need to force it to check the message exists:

```suggestion
        if message.exists():
```